crates/hir_ty/src/traits/chalk/mapping.rs

```diff
@@ -399,6 +399,7 @@ impl ToChalk for TypeCtor {
                 // this should not be reached, since we don't represent TypeName::Error with TypeCtor
                 unreachable!()
             }
+            _ => unimplemented!(), // FIXME
```

Comment:
It might be better to name all the variants here, so we know what we're missing.

Reply:
Good call - implemented